### PR TITLE
feat(button-modal): add spec of ```position``` for popup

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description

To use button-modal as popup, it is needed `position` attribute
If `position` prop is served, set button-modal `position: absolute; left: {position}rem; right: {position}rem`

스크린샷 2022-08-04 오후 7 09 18

* On mobile, `left` is always 0.75rem when `position` prop is served

### CODE REVIEW

File: src/feedbacks/modals/button-modal/PButtonModal.vue [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <template>
     <section class="p-button-modal">
         <transition v-if="visible" name="modal">
-            <div class="modal-mask" :class="{'no-backdrop':!backdrop}">
+            <div class="modal-mask"
+                 :class="[{'no-backdrop':!backdrop}, {'position': position}]"
```
Comment (yuda110) [inline src/feedbacks/modals/button-modal/PButtonModal.vue:5]: 👀
```suggestion
                 :class="[{'no-backdrop':!backdrop}, {'position': !!position}]"
```

Comment (minseolee) [reply src/feedbacks/modals/button-modal/PButtonModal.vue:5]: Oh, I didn't expect it. Thanks

File: src/feedbacks/modals/button-modal/PButtonModal.vue [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+                 :style="position ? [{'top': `${position}rem`}, {'left': `${position}rem`}] : {}"
```
Comment (yuda110) [inline src/feedbacks/modals/button-modal/PButtonModal.vue:6]: 음...props를 `position`이 아니라 `margin`이라고 하는 게 어떨까요?
position이 기존 css에서 relative, absolute 등의 값을 가지는지라 많이 헷갈릴 것 같아요.

그리고 위의 클래스명도 `position`이라기보단 `absolute`나...뭔가 커스텀된 클래스라는 것을 알 수 있었으면 좋겠습니다 :)

**Review by yuda110 — APPROVED** [review]
left a few comments :)

**Pull request merged**